src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ShardStateMetricsSnapshot.java

```diff
@@ -0,0 +1,158 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.reader;
```

Comment:
Missing License information from beginning of file.

Reply:
Added